source/frontend/src/assets/scss/colors.scss

```diff
@@ -1,23 +1,45 @@
 @import '@/../node_modules/@bcgov/design-tokens/css/variables.css';
 
+$pims-blue-20: #d9eaf7;
+$pims-blue-80: #1a5496;
+
 $pims-blue-50: #428bca;
-$pims-grey-80: #606060;
+
 $pims-yellow-10: #fef1d8;
 $pims-yellow-30: #fcba19;
 $pims-red-10: #f2dede;
+$pims-red-80: #a12622;
 $pims-white: #f2f2f2;
-$pims-blue-20: #d9eaf7;
 $pims-yellow-100: #6c4a00;
 
+$pims-grey-10: #e8ecef;
```

Comment:
I'm hesitant to introduce our own greys when grey 10 to 100 exists in the bcgov standard spec already:
https://github.com/bcgov/design-system/blob/main/packages/design-tokens/dist/css/variables.css

Reply:
That is reasonable. I added it like that because the figma spec used the code instead of a color name. The rest is the same as the response above.